src/diff-match-patch.class.ts

```diff
@@ -457,8 +457,8 @@ export class DiffMatchPatch {
    */
   public patch_make(
     a: string | Diff[],
-    opt_b: string | Diff[],
-    opt_c: string | Diff[],
+    opt_b: string | Diff[] | undefined,
```

Comment:
Shall we change these to optional arguments so that we can call `patch_make` with 1 or 2 parameters (e.g. `patch_make(text1)`) and avoid requiring explicit `undefined` values being passed (i.e. `patch_make(text1, undefined, undefined)`)?
```suggestion
    opt_b?: string | Diff[],
```